src/lib/hooks/useDistanceToUser.ts

```diff
@@ -11,21 +11,29 @@ interface UseDistanceToUserReturnType {
   getDistanceToUser: (pointA: PointType) => number | undefined
 }
 
+export function getDistanceBetweenTwoPoints(
```

Comment:
I wonder if that function could be refactored a bit: as the name suggests any two points could be put in. How about renaming the variables within accordingly and getting rid of references to user and facility? That could be taken one step further by moving the function somewhere more general-map-related such as `mapUtil.ts`?

Reply:
I like your suggestion! I'll do this!